common/spdat.cpp

```diff
@@ -431,6 +431,16 @@ bool IsCharmSpell(uint16 spell_id)
 	return IsEffectInSpell(spell_id, SE_Charm);
 }
 
+bool IsResurrectionSpell(uint16 spell_id) {
```

Comment:
Isn’t this more a “Resurrection Sickness” spell than just a “Resurrection” spell.

Reply:
`IsResurrectionSicknessSpell`.